bin/i18n/sync-in.rb

```diff
@@ -101,18 +101,11 @@ def get_i18n_strings(level)
         i18n_strings['behavior_names'][name.content] = name.content if name
       end
 
-      text_blocks = blocks.xpath("//block[@type=\"text\"]")
-      i18n_strings['placeholder_texts'] = Hash.new unless text_blocks.empty?
-      text_blocks.each do |text_block|
-        text_title = text_block.at_xpath('./title[@name="TEXT"]')
-        # Skip empty or untranslatable string.
-        # A translatable string must have at least 3 consecutive alphabetic characters.
-        next unless text_title&.content =~ /[a-zA-Z]{3,}/
-
-        # Use only alphanumeric characters in lower cases as string key
-        text_key = Digest::MD5.hexdigest text_title.content
-        i18n_strings['placeholder_texts'][text_key] = text_title.content
-      end
+      ## Placeholder texts
+      i18n_strings['placeholder_texts'] = Hash.new
+      i18n_strings['placeholder_texts'].merge! get_placeholder_texts(blocks, 'text', ['TEXT'])
+      i18n_strings['placeholder_texts'].merge! get_placeholder_texts(blocks, 'studio_ask', ['TEXT'])
+      i18n_strings['placeholder_texts'].merge! get_placeholder_texts(blocks, 'studio_showTitleScreen', %w[TEXT TITLE])
```

Comment:
I don't think it matters, but do note that

```suggestion
      i18n_strings['placeholder_texts'].merge! get_placeholder_texts(blocks, 'studio_showTitleScreen', %w(TEXT TITLE))
```

is more standard. Consistency would also be nice:

```ruby
      i18n_strings['placeholder_texts'].merge! get_placeholder_texts(blocks, 'text', %w(TEXT))
      i18n_strings['placeholder_texts'].merge! get_placeholder_texts(blocks, 'studio_ask', %w(TEXT))
      i18n_strings['placeholder_texts'].merge! get_placeholder_texts(blocks, 'studio_showTitleScreen', %w(TEXT TITLE))
```